WORKSPACE

```diff
@@ -69,17 +69,5 @@ register_unittest_toolchains()
 # github.com/bazelbuild/rules_go is available from io_bazel_rules_go and it
 # doesn't need to duplicatively fetch it.
 # gazelle:repository go_repository name=io_bazel_rules_go importpath=github.com/bazelbuild/rules_go
```

Comment:
I'm not familiar with gazelle repository hints, do these need to be moved to gazelee/WORKSPACE?

Reply:
I've moved them, and everything still seems to build. Apparently, it's fine to move them.